docs/topics/circuit-breaker-mission-access-hystrix-dashboard-spring-boot.adoc

```diff
@@ -0,0 +1,57 @@
+= Using the Hystrix Dashboard to Monitor the Circuit Breaker
+
+Hystrix Dashboard lets you easily monitor the health of your services in real time by aggregating Hystrix metrics data from an event stream and displaying them on one screen. For more detail, see the Hystrix Dashboard link:https://github.com/Netflix/Hystrix/wiki/Dashboard[wiki page].
+
+NOTE: You must have the Circuit Breaker booster application deployed before proceeding with the steps below.
+
+//  Openshift first. Do we reference Minishift to? Add a note or a section?
+// Replace with proper OSO URLs
+. Log in to your {OpenShiftOnline} instance.
++
+[source,bash,subs="attributes+"",options="nowrap"]
+--
+oc login https://192.168.42.158:8443 -u developer -p developer
+--
++
+Alternatively, you can log in using the web console. You can access it by opening your OpenShift URL in your browser or by executing the following  command.
++
+[source,bash,subs="attributes+"",options="nowrap"]
+--
+oc console
```

Comment:
Thanks, I missed this, You're absolutely right, there is no such command, should be `minishift console`:) fixing